pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: clever_ads_solutions
 description: Clever Ads Solutions Flutter plugin
-version: 0.3.1
+version: 0.3.2
```

Comment:
Upgrade the plugin version to 0.4.0.  
Also set description to `The CAS.AI Flutter plugin allows you to easily monetize your apps for iOS and Android using lot of Ads SDKs in mediation.`

Reply:
Set homepage to `https://github.com/cleveradssolutions/CAS-Flutter`